api/src/main/java/module-info.java

```diff
@@ -372,51 +372,51 @@
  *     match the name of an entity attribute and the repository must be compiled
  *     with the {@code -parameters} compiler option so that parameter names are
  *     available at runtime. For example,
- *     <pre>
- *     &#64;Find
- *     Optional&lt;Product&gt; get(&#64;By("id") long productId);
+ *     <pre>{@code
+ *     @Find
+ *     Optional<Product> get(@By("id") long productId);
  *
- *     &#64;Find
- *     &#64;OrderBy("price")
- *     List&lt;Product&gt; discounted(
- *             &#64;By("discount") &#64;Is(AtLeast.class) float minAmount);
+ *     @Find
+ *     @OrderBy("price")
+ *     List<Product> discounted(
+ *             @By("discount") @Is(AtLeast.class) float minAmount);
  *
- *     &#64;Find
- *     &#64;OrderBy("price")
+ *     @Find
+ *     @OrderBy("price")
  *     Product[] named(String name);
- *     </pre>
+ *     }</pre>
  * </li>
  * <li>The parameter is a {@link Constraint} or a
  *     {@linkplain jakarta.data.constraint subtype} of {@code Constraint} and
  *     has exactly the same name (per the rule from category 1) as an attribute
  *     of the entity class. The constraint must be parameterized with the same
  *     type (or if primitive, the corresponding wrapper type) as the attribute.
  *     For example,
- *     <pre>
- *     &#64;Delete
- *     int discontinue(&#64;By("id") In&lt;Long&gt; productIds);
- *
- *     &#64;Find
- *     &#64;OrderBy("price")
- *     &#64;OrderBy("name")
- *     Stream&lt;Product&gt; pricedUnder(LessThan&lt;Float&gt; price);
- *     </pre>
+ * <pre>{@code
+ *     @Delete
+ *     int discontinue(@By("id") In<Long> productIds);
+ *
+ *     @Find
+ *     @OrderBy("price")
+ *     @OrderBy("name")
+ *     Stream<Product> pricedUnder(LessThan<Float> price);
+ * }</pre>
  * </li>
  * <li>The parameter is a {@link Restriction} and its type parameter is the
  *     entity class. For example,
- *     <pre>
- *     &#64;Find
- *     List&lt;Product&gt; search(Restriction&lt;Product&gt; restrict);
- *     </pre>
+ * <pre>{@code
+ *     @Find
+ *     List<Product> search(Restriction<Product> restrict);
+ * }</pre>
  * </li>
  * <li>The parameter is a {@link Limit}, {@link Sort}, {@link Order}, or
  *     {@link PageRequest} (discussed under the section titled
  *     <em>Special parameters</em>) and the repository method is annotated with
  *     {@link Find}. For example,
- *     <pre>
- *     &#64;Find
- *     Page&lt;Product&gt; getPage(PageRequest pageRequest, Order&lt;Product&gt; sortBy);
- *     </pre>
+ *     <pre>{@code
+ *     @Find
+ *     Page<Product> getPage(PageRequest pageRequest, Order<Product> sortBy);
+ *    }</pre>
```

Comment:
To match indentation above
```suggestion
 *     }</pre>
```